content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+Because "s3-test-iam_user" is inside account - "s3test", the Bucket will be also placed under account - "s3test".
+
+To list Buckets using the `mc` command:
+
+```shell
+/opt/mapr/bin/mc ls --account s3test --versions --recursive --summarize --json s3-test-iam_user-alias
+```
+
+Sample output:
+
+```json
+{
+ "status": "success",
+ "type": "folder",
+ "lastModified": "2023-02-16T15:22:16+08:00",
+ "size": 23897893980,
+ "key": "s3-test-iam-user-bucket/",
+ "etag": "",
+ "url": "https://m2-maprts-vm197-172.mip.storage.hpecorp.net:9000/",
+ "versionOrdinal": 1
+}
+{
+"totalObjects": 1,
+"totalSize": 23897893980
+}
+```
+
+#### Install the AWS CLI and put an file into the Bucket
+
+To install the AWS CLI, refer to this Amazon AWS document 👉 [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+Then you create a profile for the IAM User:
+
+```shell
+export AWS\_CA\_BUNDLE=/opt/mapr/conf/ca/chain-ca.pem
+aws configure --profile s3-test-iam_user-ray-2-objstor
+```
+
+❗Note: Before using the AWS CLI, remember to export the environment AWS\_CA\_BUNDLE=/opt/mapr/conf/ca/chain-ca.pem.
+Otherwise AWS CLI cannot communicate with S3server because S3server is using self-signed TLS certificates.
+
+After inputting the above command, the AWS CLI will ask you to input the access key and secret key.
+After the profile is created, the information will be stored at <ins>$HOME/.aws/config</ins> and <ins>$HOME/.aws/credentials</ins>.
+
+Use the below command to list Buckets:
+
+```shell
+aws s3api list-buckets --endpoint-url https://`hostname -f`:9000 --profile s3-test-iam_user-ray-2-objstor
+```
+
+Use the below command to put a file into the Bucket:
+
+```shell
+aws s3api put-object --bucket s3-test-iam-user-bucket --key 'testdir/s3-test-iam-user-dir/hpe-cp-rhel-release-5.5.1-3083.bin' --body 'downloads/hpe-cp-rhel-release-5.5.1-3083.bin' --endpoint-url https://m2-maprts-vm197-172.mip.storage.hpecorp.net:9000 --profile s3-test-iam_user-ray-2-objstor
+```
+
+🗒Note: "s3-test-iam-user-bucket" is the Bucket's name that you created before.
+"testdir/s3-test-iam-user-dir/hpe-cp-rhel-release-5.5.1-3083.bin" is the path that you want to put into the Bucket.
+The part of "testdir/s3-test-iam-user-dir/" indicates it's under this directory, if the directory doesn't exist, it will be created.
+"downloads/hpe-cp-rhel-release-5.5.1-3083.bin" is the local file path that you want to put into the Bucket.
+
+## Create an Cold-Tiered Volume and offlad to remote Object Store
+
+So, now we are going to create a Volume on another cluster and configure the Cold Tier remote target for this Volume.
+Then we will manually offload the data in this Volume to the remote HPE Ezmeral Data Fabric Object Store.
+
+### Create a Cold-Tiered Volume by Web GUI
+
+First of all, you need to log in to MCS, and enter the following positions in turn at the top of the screen: Data --> Volumes👇
```

Comment:
Modify to read "First, you will need to log into MCS, and... (note "into" as well as rewording)